klt-cert-manager/Makefile

```
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= klc-certmanager:latest
+IMG ?= klt-certmanager:latest
```

no `-` in `certmanager`?
```suggestion
IMG ?= klt-cert-manager:latest
```